src/MediaCodecCryptoInfo.cpp

```diff
@@ -132,3 +132,10 @@ void CJNIMediaCodecCryptoInfo::set(int newNumSubSamples,
   env->DeleteLocalRef(Key);
   env->DeleteLocalRef(IV);
 }
+
+void CJNIMediaCodecCryptoInfo::setPattern(const CJNIMediaCodecCryptoInfoPattern &pattern)
+{
+  call_method<void>(m_object, "setPattern",
+    "(Landroid/media/MediaCodec$CryptoInfo$Pattern;)V",
+    pattern.get_raw());
```

Comment:
This needs to be encased in a version check, as setPattern was introduced in SDK 24.